src/jit/compiler.cpp

```diff
@@ -2055,7 +2055,9 @@ void                Compiler::compInitOptions(unsigned compileFlags)
             static ConfigMethodSet fJitDisasm;
             fJitDisasm.ensureInit(CLRConfig::INTERNAL_JitDisasm);
             if (fJitDisasm.contains(info.compMethodName, info.compClassName, info.compMethodInfo->args.pSig))
+            {
```

Comment:
This matches our current coding conventions, but leaves everything else alone. It seems unrelated to your change. I wouldn't touch it unless you need to.